docs/index.html

```diff
@@ -76,7 +76,6 @@
         rows[i].onclick = function () {
             var cssClass = this.getAttribute( 'class' );
             var iconID = cssClass.split(' gridicons-')[1];
-			alert(iconID);
```

Comment:
I know there were recent changes to the gridicons page, it was probably related to that. Probably fine to let the grunt task to its thing.